packages/framework/src/Component/Grid/Exception/OrderingNotSupportedException.php

```diff
@@ -6,7 +6,7 @@
 
 use Exception;
 
```

Comment:
what about those interfaces? they were left intentionaly?

![Snímek obrazovky 2025-01-13 v 16 12 31](https://github.com/user-attachments/assets/570d3891-fdd9-4ada-ba30-39d97e460255)

Reply:
- `MailException` was forgotten, removed in cd48270491
-  `FormException` removed in ab00a6b96a
- `PerformanceException` removed in 52baacd569

Please let's discuss once more the following - they are used in catch blocks, and some of them might be replaced by regular common exception classes (but some of them probably not - e.g. there are PromoCode exceptions that extend NotFoundException)...

- `FileUploadException`
- `ImageConfigException`
- `MoneyException`
- `UploadedFileConfigException`
- `PromoCodeException`
- `VariantException`